simplehub_test.go

```diff
@@ -0,0 +1,210 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package pubsub_test
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/juju/testing"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/pubsub"
+)
+
+type SimpleHubSuite struct {
+	testing.LoggingCleanupSuite
+}
+
+var (
+	_ = gc.Suite(&SimpleHubSuite{})
+
+	veryShortTime = time.Millisecond
```

Comment:
What I mean is that for something that should take 5ns, 1ms is a very long time. Why wait for longer for something not to happen?